### DISCUSSION TITLE
Array/model length

### DISCUSSION THREAD

**Discussion by MaximkaHuman** [slint-ui/slint#519] [Q&A] [answered]
Hello! The main problem is setup ScrollView _viewport-height_ property according to array/model length. Is it possible to get array/model length via .60?

**Comment by ogoffart** [comment, marked as answer]
This is currently not possible in .60
We plan to implement that, see https://github.com/sixtyfpsui/sixtyfps/issues/98
In the mean time, one would have to use a callback imlemented in native code.

**Reply by MaximkaHuman** [reply]
Thanks @ogoffart
